Assets/MixedRealityToolkit.ThirdParty/OculusQuestInput/Art/Shaders/GradientSkybox.shader

```diff
@@ -0,0 +1,86 @@
+﻿/************************************************************************************
+
+Copyright (c) Facebook Technologies, LLC and its affiliates. All rights reserved.  
+
+See SampleFramework license.txt for license terms.  Unless required by applicable law 
+or agreed to in writing, the sample code is provided “AS IS” WITHOUT WARRANTIES OR 
+CONDITIONS OF ANY KIND, either express or implied.  See the license for specific 
+language governing permissions and limitations under the license.
```

Comment:
If you could add your changes to the disclaimer, header, that would be appreciated.
If you could also point to the directory of the original, that would be great.

Reply:
I've updated the header with the requested information.